### ISSUE TITLE
adding a blog

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-anthology#722]
I'd like to add a blog to the Anthology. It would simplify a lot of reporting (such as when volumes are added) and might supersede #358, via the use of scripts such as that in #711 which can generate reports based on git diffs.

I haven't looked into this much yet but imagine it's easy with Hugo.

**Comment by mjpost** [comment]
It would also be useful to be able to host and post links to reports and things of that nature. The ACL AdminWiki could serve for this but some of it has a natural home here.

**Comment by knmnyn** [comment]
I agree.  This should be pretty easy to do given that Hugo behaves somewhat like Jekyll and other static templates that evolved to do blogging.  I feel that the AdminWiki is a good place to duplicate or summarize such content but a blog vehicle would be a great way to show that the Anthology is actively worked on.

**Comment by mbollmann** [comment]
What functionality would this need, besides what we already have in the old and hidden-away posts at https://aclanthology.org/posts/ ?

I imagine if we made the list page slightly more informative (with date + preview of the post) and linked it from the top menu, this would essentially already serve as a blog?

**Comment by mjpost** [comment]
Yes, good point, this page would work fine. We'd also need to add an RSS feed.

**Comment by mbollmann** [comment]
This exists now: https://aclanthology.org/posts/
